tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/TestContext.cs

```diff
@@ -206,6 +210,7 @@ private class Scope
 
             public Scope(IRequestHandler<GetSolutionByIdQuery, ISolution> getSolutionByIdHandler,
                 IRequestHandler<GetClientApplicationBySolutionIdQuery, IClientApplication> getClientApplicationBySolutionIdHandler,
+                IRequestHandler<GetHostingBySolutionIdQuery, IHosting> getHostingHandler,
```

Comment:
Could this variable match the name of the property e.g. getHostingBySolutionIdHandler?